FinniversKit/Sources/Components/SelectionListView/Subviews/SelectionListItemView.swift

```diff
@@ -115,17 +120,20 @@ class SelectionListItemView: UIView {
             textStackView.leadingAnchor.constraint(equalTo: selectionView.trailingAnchor, constant: .spacingM),
             textStackView.bottomAnchor.constraint(equalTo: detailViewsStackView.topAnchor, constant: -.spacingM),
 
-            titleLabel.widthAnchor.constraint(equalTo: descriptionLabel.widthAnchor),
-
```

Comment:
Maybe a lot to ask, but do you recall the point of this constraint? Am I breaking something here? @bstien

Reply:
No, that line was added in PR #1144. I don't know if it'll break anything. Do you, @AtishPapa?

Reply:
Honestly I don't remember why this was added. 🤔